dds_cli/__init__.py

```diff
@@ -61,12 +61,15 @@ class DDSEndpoint:
     BASE_ENDPOINT_DOCKER = "http://dds_backend:5000/api/v1"
     BASE_ENDPOINT_REMOTE = "https://delivery.scilifelab.se/api/v1"
     BASE_ENDPOINT_REMOTE_TEST = "https://testing.delivery.scilifelab.se/api/v1"
+    BASE_ENDPOINT_REMOTE_DEV = "https://dev.delivery.scilifelab.se/api/v1"
     if os.getenv("DDS_CLI_ENV") == "development":
         BASE_ENDPOINT = BASE_ENDPOINT_LOCAL
     elif os.getenv("DDS_CLI_ENV") == "docker-dev":
         BASE_ENDPOINT = BASE_ENDPOINT_DOCKER
     elif os.getenv("DDS_CLI_ENV") == "test-instance":
         BASE_ENDPOINT = BASE_ENDPOINT_REMOTE_TEST
+    elif os.getenv("DDS_CLI_ENV") == "development-instance":
```

Comment:
picky but I'd prefer this: 
```suggestion
    elif os.getenv("DDS_CLI_ENV") == "dev-instance":
```
Change this in the same push as the version